flow/util/genMetrics.py

```diff
@@ -438,7 +420,7 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
         f.write(all_df.to_html())
 else:
     metrics_dict, metrics_df = extract_metrics(
-        args.flowPath,
+        os.getcwd(),
```

Comment:
How does this work?  You removed the chdir so cwd could be anywhere.

Reply:
All the ORFS example designs assume that FLOW_HOME is cwd, they use relative paths.

With this change, make metadata works for a cwd outside of FLOW_HOME.